libs/sdk-backend-spi/src/workspace/index.ts

```diff
@@ -176,6 +176,12 @@ export interface IWorkspacesQuery {
      */
     withOffset(offset: number): IWorkspacesQuery;
 
+    /**
+     * Sets a identifier of the parent workspace to get its children, otherwise the root workspace.
+     * @param workspaceId - identifier of the parent workspace
+     */
+    withParent?(workspaceId: string | undefined): IWorkspacesQuery;
```

Comment:
Things on the SPI should not be optional, that would make using them tricky. Instead they should throw some error on backends that do not support such feature.

Reply:
More specificaly it should throw `NotSupported` from `@gooddata/sdk-backend-spi`

Reply:
@no23reason Thank you for your comments! I have updated this. Please help to verify the changes again. Thanks!

Reply:
@no23reason , could you please review this PR? Thank you!